Sources/TokamakDOM/Views/Buttons/Button.swift

```diff
@@ -17,10 +17,25 @@
 
 import TokamakCore
 
-extension Button: ViewDeferredToRenderer where Label == Text {
+extension _Button: ViewDeferredToRenderer where Label == Text {
   public var deferredBody: AnyView {
-    AnyView(HTML("button", listeners: ["click": { _ in _ButtonProxy(self).action() }]) {
-      _ButtonProxy(self).label.subject
+    AnyView(HTML("button", listeners: [
+      "click": { _ in action() },
+      "onmousedown": { _ in isPressed = true },
+      "onmouseup": { _ in isPressed = false },
```

Comment:
```suggestion
      "onpointerdown": { _ in isPressed = true },
      "onpointerup": { _ in isPressed = false },
```

This still doesn’t work with keyboard input (ie press space to press the button, release space to unpress) but it will work with touch